recipe/migrations/geotiff171.yaml

```diff
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+migrator_ts: 1648226691.6519449
+geotiff:
+- '1.7.1'
```

Comment:
Did the ABI change?

Reply:
Can you send a separate PR for the `conda_build_config.yaml` change and we can think about the migrator in this PR?